resources/CollectiblesRebirth.xml

```diff
@@ -1,5 +1,8 @@
 <data>
   <group Name="Collect">
+    <entitydefaults>
+      <tag>InNonCombatRooms</tag>
+    </entitydefaults>
```

Comment:
It looks like it's not used in BR's xmls currently, but entitydefaults are also able to specify other default properties.

For example, `<entitydefaults ID="5"/>` would cause all entities within a group to default to an ID of 5 if not specified.

With that in mind, maybe the `<defaults>` tag concept would work best?